src/test/jtx/Env.h

```diff
@@ -120,6 +120,19 @@ class Env
 
     Account const& master = Account::master;
 
+    /// Used by parseResult() and postConditions()
+    struct ParsedResult
+    {
+        TER ter;
+        // One way that RPC errors are returned
+        error_code_i rpcCode = rpcSUCCESS;
+        std::string rpcMessage;
+        // Another way that RPC errors are returned
```

Comment:
I'm of two minds about the two comments about ways that RPC errors are returned.  When I saw the comments it felt like they were crying out for a discriminated `union`.  But I think the current implementation is fine, and I don't see a good reason to cobble together a union.

So I'm not sure the comments are truly enlightening, or if they are more of a distraction.

Reply:
I rewrote the comments.